vocode/streaming/synthesizer/azure_synthesizer.py

```diff
@@ -364,16 +364,19 @@ def ready_synthesizer(self):
     def get_message_up_to(
         self,
         message: str,
-        ssml: str,
+        ssml_parts: list[str],
         seconds: float,
         word_boundary_event_pool: WordBoundaryEventPool,
     ) -> str:
+        if len(ssml_parts) > 1:
+            return message
+        else:
+            ssml = ssml_parts[0]
```

Comment:
convenience

Reply:
python will let you use the `ssml` variable later even though it's inside an `else` block?

Reply:
i guess it does. Don't need the else block so I'll remove it.

Reply:
It was not my intention to have the try outside of the else block. Removing the else all together is cleaner